assembl/static2/js/app/pages/idea.jsx

```diff
@@ -16,6 +16,50 @@ import ColumnsView from '../components/debate/multiColumns/columnsView';
 import ThreadView from '../components/debate/thread/threadView';
 import { DeletedPublicationStates } from '../constants';
 
+const deletedPublicationStates = Object.keys(DeletedPublicationStates);
+
+const sortByCreationDateDesc = (a, b) => {
```

Comment:
`comparator` in the name would make it more clear what this is

Reply:
ok